local-dev-first-start-only.sh

```diff
@@ -269,7 +269,8 @@ sed -i $SED_EXT 's!SENDFILE_IRODS_USER!'${SENDFILE_IRODS_USER}'!g' nginx/Dockerf
 sed -i $SED_EXT 's!SENDFILE_IRODS_GROUP!'${SENDFILE_IRODS_GROUP}'!g' nginx/Dockerfile-nginx
 
 cp hydroshare/local_settings.template hydroshare/local_settings.py 2>/dev/null
-mkdir -p hydroshare/static/media 2>/dev/null
+mkdir -p hydroshare/static 2>/dev/null
```

Comment:
do we also need to mkdir hysroshare/static/static ?

Reply:
Yeah we need that too.